samples/CoreAnimationTest/CoreAnimationTest/ProgrammaticViewController.m

```diff
@@ -38,13 +38,13 @@ - (void)viewDidLoad {
     [self addMenuItemViewController:[[CALayerAnimationViewController alloc] init] andTitle:@"CALayer Animation (CAAnimation)"];
 
     // CAShapeLayer
-    [self addMenuItemSegue:@"CAShapeLayer" andTitle:@"CAShapeLayer"];
+    [self addMenuItemSegue:@"CAShapeLayer" andTitle:@"CAShapeLayer - not supported"];
```

Comment:
Partially - most of the controls in this VC work fine except for a few. If < ~50% was non-functional, we'd remove the VC from the sample.

---
In reply to: [99437888](https://github.com/Microsoft/WinObjC/pull/1874#discussion_r99437888) [](ancestors = 99437888)